src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -196,6 +359,39 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class DescendantToken {
+ *   public void exampleMethod() {
+ *       try {
+ *           // Some code
+ *       } catch (Exception e) { // OK
+ *           System.out.println("Caught exception");
+ *           return;
+ *       } finally { // OK
+ *           System.out.println("Finally block");
+ *
+ *       }
+ *       try {
+ *           // Some code
+ *       } catch (Exception e) {
+ *           try { // Violation
+ *               // Some code
+ *           } catch (Exception ex) {
+ *               // handle exception
+ *           }
+ *       } finally {
+ *           try { // Violation
```

Comment:
because in this example we are trying to produce a violation on a try statement within a catch or finally block. so i tried to show violation on both "try statement within a catch block" and "try statement within a finally block". That is why it has two violations.

Reply:
I got it